tools/android-store/Makefile

```diff
@@ -30,6 +29,10 @@ PRIVATE_KEY ?= ../$(APP_NAME)-private-key.pem
 PUBLIC_CERT ?= ../$(APP_NAME)-signing-cert.pem
 WRAP_KEY ?= ../encryption_public_key.pem
 
+# The user and group id for the app user
+DOCKER_USER_ID = 10001
```

Comment:
Magic numbers from our [Dockerfile](https://github.com/mozilla-services/autograph/blob/main/Dockerfile#L75)

Reply:
Oh, I meant write this comment in the file itself.